flowcraft/generator/templates/stx_seqtyping_reads.nf

```diff
@@ -0,0 +1,66 @@
+stx2covered = ""
+stx2identity = ""
```

Comment:
This two empty string would be more explicit if they are in the else statement of the `if`s below.

Reply:
I'll change it

Reply:
Done